samples/core/ai_platform/ai_platform.ipynb

```diff
@@ -345,4 +345,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 2
-}
\ No newline at end of file
+}
```

Comment:
Do you mind removing the new \n in the end?

Reply:
I mean adding back.

Reply:
Got it, I edited with GitHub file editor, looks like it does that by default.